File: ProChat/ProChat/Datos/Archivo.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProChat.Datos
{
    public class Archivo
    {
        public int id { get; set; }
        public string name { get; set; }
        public string contentType { get; set; }
        public int from { get; set; }
        public int to { get; set; }
        public string date { get; set; }
    }
}

File: Moviles/Proyecto/app/src/main/java/Datos/Archivo.java
package Datos;

/**
 * Created by Alejandro on 11/04/2016.
 */
public class Archivo {
    private Integer id;
    private String name;
    private String contentType;
    private Integer from;
    private Integer to;
    private String date;

    public Integer getFrom() {
        return from;
    }

    public Integer getId() {
        return id;
    }

    public Integer getTo() {
        return to;
    }

    public String getContentType() {
        return contentType;
    }

    public String getDate() {
        return date;
    }

    public String getName() {
        return name;
    }
}
